### PR TITLE
Set verbose log output to debug

### PR DESCRIPTION
**Pull request by ryancouto**
and set default level to info instead of returning on error parsing

### CODE REVIEW

File: binaries/apiserver/main.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func main() {
 	level, err := log.ParseLevel(*logLevelFlag)
 	if err != nil {
 		log.Error(err)
-		return
+		level = log.InfoLevel
```
Comment (jasonbs10) [inline binaries/apiserver/main.go]: Why do we want to continue starting up if there's an error parsing the level? Startup is the perfect time to bomb due to errors.

Comment (JeanetteBruno) [inline binaries/apiserver/main.go]: +1.   Also - does the logrus Fatal trigger a call to os.Exit(1) (as the log package does?)   The logrus docs just say it logs a message at the Fatal level doesn't mention os.Exit(1).... the next question is do we want it to exit?

Comment (ryancouto) [inline binaries/apiserver/main.go]: Reverting. And it does Exit(1), 
https://github.com/sirupsen/logrus/blob/master/entry.go#L164
https://github.com/sirupsen/logrus/blob/547e984ad93a90192134fc0fdb57a468edb514fe/alt_exit.go#L49

**Comment by JeanetteBruno** [comment]
lgtm

File: ice/eval.go [hunk 2]
```diff
@@ -36,8 +36,8 @@ func (bag *MagicBag) Extract(dest interface{}) (result error) {
 	targetVal.Set(reflect.Value(eval.construct(targetType)))
 
 	s := spew.Sdump(dest)
-	log.Info("************* Extracted:")
-	log.Info(s)
+	log.Debug("************* Extracted:")
```
Comment (jasonbs10) [inline ice/eval.go]: I deleted this in my latest PR so we should just drop this change.

**Pull request closed without merging**